cni/deployments/kubernetes/install/test/install_cni.go

```diff
@@ -247,21 +296,36 @@ func doTest(testNum int, wd, preConfFile, resultFileName, expectedOutputFile,
 		docker("logs", containerID, t)
 		docker("rm", containerID, t)
 	}()
-	time.Sleep(10 * time.Second)
 
-	compareConfResult(testWorkRootDir, tempCNIConfDir, resultFileName, expectedOutputFile, t)
+	resultFile := tempCNIConfDir + "/" + resultFileName
+	if delayedConfFile != "" {
+		timeout := time.After(5 * time.Second)
+		tick := time.Tick(500 * time.Millisecond)
```

Comment:
It would be nice to use a `time.NewTicker()` and `defer ticker.Stop()`, to avoid leaking `Tick`s and goroutines.